src/client/style/Profile.css

```diff
@@ -90,16 +90,45 @@
 
 
 .profile-modal-section-header {
-  font-family: 'Merriweather', Times, serif;
+  font-family: 'Open Sans', Arial, sans-serif;
   font-size:  1rem;
-  margin-bottom: 6px;
 }
 
 .profile-verified-yes {
-  font-family: 'Merriweather', Times, serif;
+  font-family: 'Open Sans', Arial, sans-serif;
   font-size:  1rem;
 }
 
+.profile-user-image-display{
+  margin: auto;
+  height: 8em;
+  width: 8em;
+  border: 1px solid black;
+}
+
+.profile-user-image-edit-button{
+  background: none!important;
```

Comment:
I think using "!important" is considered bad practice because it makes debugging CSS in the future more difficult

Reply:
removed !important #197, please review & approve that PR